packages/core-knapsack/tokens.ts

```diff
@@ -0,0 +1,50 @@
+import { readFileSync, mkdirSync, writeFileSync } from "fs";
+import { join } from "path";
+import { execSync } from "child_process";
+import merge from "deepmerge";
+
+// The "sets" within Tokens Studio we want to combine
+const combos = [
```

Comment:
Are we expecting this repo to be used as a generic demo? If so, might want different names than `GSK`, etc.